### PR TITLE
Add config output-handle-key/output-old-value doc for ticdc

### PR DESCRIPTION
**Pull request by hongyunyan**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/16874
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by sdojjy — APPROVED** [review]

File: ticdc/ticdc-changefeed-config.md [hunk 1]
```diff
@@ -193,6 +193,12 @@ enable-partition-separator = true
 # include-commit-ts = false
 # The encoding method of binary data, which can be 'base64' or 'hex'. The default value is 'base64'.
 # binary-encoding-method = 'base64'
+# Whether to output handle key information. The default value is false. 
+# This variable is for internal implementation only and setting this variable is not recommended.
```
Comment (lilin90) [inline ticdc/ticdc-changefeed-config.md:197]:
```suggestion
# This configuration parameter is for internal implementation only and it is not recommended to set it.
```

File: ticdc/ticdc-changefeed-config.md [hunk 2]
```diff
@@ -193,6 +193,12 @@ enable-partition-separator = true
...
+# output-handle-key = false
+# Whether to output the value before the row data changes. The default value is false. 
+# After being turned on, the Update event will output two lines of data. The first line is a Delete event, which outputs the data before the change, and the second line is a Insert event, which outputs the changed data.
```
Comment (lilin90) [inline ticdc/ticdc-changefeed-config.md:200]:
```suggestion
# When it is enabled, the UPDATE event will output two rows of data: the first row is a DELETE event that outputs the data before the change; the second row is an INSERT event that outputs the changed data.
```

**Review by lilin90 — APPROVED** [review]
/lgtm

**Pull request merged**